Integrations/ThreatConnect/ThreatConnect.yml

```diff
@@ -0,0 +1,2011 @@
+      name: tag
+      required: false
+      secret: false
+    deprecated: false
+    description: Create a ThreatConnect group from type Event
+    execution: false
+    name: tc-create-event
+    outputs:
+    - contextPath: TC.Event.Name
+      description: Event name
+      type: string
+    - contextPath: TC.Event.Date
+      description: Event Date
+      type: date
+    - contextPath: TC.Event.Status
+      description: Event status
+      type: string
+    - contextPath: TC.Event.Owner
+      description: Event's owner
+      type: string
+    - contextPath: TC.Event.Tag
+      description: Event's tag
+      type: string
+    - contextPath: TC.Event.ID
+      description: Event ID
+      type: string
+  - arguments:
+    - default: false
+      description: Threats group name
+      isArray: false
+      name: name
+      required: true
+      secret: false
+    deprecated: false
+    description: Create a ThreatConnect group from type Threats
+    execution: false
+    name: tc-create-threat
+    outputs:
+    - contextPath: TC.Threat.Name
+      description: Threat's name
+      type: string
+    - contextPath: TC.Threat.ID
+      description: Threat's ID
+      type: string
+  - arguments:
+    - default: false
+      description: Group's ID
+      isArray: false
+      name: groupID
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Group type (Incident,Event,Campaign,Threat)
+      isArray: false
+      name: type
+      predefined:
+      - Incidents
+      - Events
+      - Campaigns
+      - Threats
+      required: true
+      secret: false
+    deprecated: false
+    description: Delete a Group from ThreatConnect (Incident, Event, Campaign, Threat)
+    execution: false
+    name: tc-delete-group
+  - arguments:
+    - default: false
+      description: The ID of the group. To get the ID, run get-groups command
+      isArray: false
+      name: group_id
+      required: true
+      secret: false
+    - default: false
+      description: The type of the attribute
+      isArray: false
+      name: attribute_type
+      required: true
+      secret: false
+    - default: false
+      description: The attribute value
+      isArray: false
+      name: attribute_value
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: The type of the group
+      isArray: false
+      name: group_type
+      predefined:
+      - adversaries
+      - campaigns
+      - documents
+      - emails
+      - events
+      - incidents
+      - intrusionSets
+      - reports
+      - signatures
+      - threats
+      required: true
+      secret: false
+    deprecated: false
+    description: Add attribute to a sepcific event
+    execution: false
+    name: tc-add-group-attribute
+    outputs:
+    - contextPath: TC.Group.DateAdded
+      description: The date the attribute was added
+      type: Date
+    - contextPath: TC.Group.ID
```

Comment:
i think you're mixing between group id and attribute id, same for type and value

Reply:
?

Reply:
Fixed. The type and value are ok - since the path is under group but the value and type are for the attribute